spec/swagger_helper.rb

```diff
@@ -17,6 +17,12 @@
           bearer_auth: { type: :http, scheme: :bearer }
         }
       },
+      tags: [
+        {
+          name: 'User',
+          description: 'Operations about User'
```

Comment:
```suggestion
          description: 'User related actions'
```

Just sounds better imo

Reply:
fixed